search-commons/src/test/java/no/unit/nva/search2/ResourceClientTest.java

```diff
@@ -98,69 +118,159 @@ class ResourceQueries {
-                .withRequiredParameters(FROM, SIZE)
-                .withOpensearchUri(URI.create(container.getHttpHostAddress()))
+                .withOpensearchUri(hostAddress)
+                .withRequiredParameters(FROM, SIZE, AGGREGATION)
                 .build()
-                .withRequiredStatus(PublicationStatus.PUBLISHED, PublicationStatus.PUBLISHED_METADATA);
-            var response1 = searchClient.doSearch(query1);
+                .withRequiredStatus(PUBLISHED, PUBLISHED_METADATA);
             var response2 = searchClient.doSearch(query2);
-
-            assertNotNull(response1);
             assertNotNull(response2);
 
-            assertEquals(response1.aggregations(),response2.aggregations());
+            assertEquals(response1.aggregations(), response2.aggregations());
 
             var aggregations = query1.toPagedResponse(response1).aggregations();
 
             assertFalse(aggregations.isEmpty());
-            assertThat(aggregations.get(TYPE).size(), is(3));
+            assertThat(aggregations.get(TYPE).size(), is(4));
             assertThat(aggregations.get(HAS_FILE).size(), is(2));
-            assertThat(aggregations.get(HAS_FILE).get(0).count(), is(19));
+            assertThat(aggregations.get(HAS_FILE).get(0).count(), is(20));
             assertThat(aggregations.get(FUNDING_SOURCE).size(), is(2));
-            assertThat(aggregations.get(PUBLISHER).size(), is(2));
+            assertThat(aggregations.get(PUBLISHER).get(0).count(), is(8));
             assertThat(aggregations.get(CONTRIBUTOR).size(), is(12));
             assertThat(aggregations.get(TOP_LEVEL_ORGANIZATION).size(), is(4));
             assertThat(aggregations.get(TOP_LEVEL_ORGANIZATION).get(1).labels().get("nb"),
                        is(equalTo("Sikt – Kunnskapssektorens tjenesteleverandør")));
         }
 
+        @Test
+        void userSettingsNotFoundReturn200() throws IOException, InterruptedException, BadRequestException {
+            var mochedHttpClient = mock(HttpClient.class);
+            var userSettingsClient = new UserSettingsClient(mochedHttpClient, setupMockedCachedJwtProvider());
+            var mockedResponse = mockedHttpResponse(EMPTY_USER_RESPONSE_JSON, 200);
+            when(mochedHttpClient.send(any(), any()))
+                .thenReturn(mockedResponse);
+            var searchClient = new ResourceClient(HttpClient.newHttpClient(), userSettingsClient,
+                                                  setupMockedCachedJwtProvider());
+
+            var uri = URI.create("https://x.org/?CONTRIBUTOR=https://api.dev.nva.aws.unit.no/cristin/person/1136254");
+            var response = ResourceQuery.builder()
+                .fromQueryParameters(queryToMapEntries(uri))
+                .withOpensearchUri(URI.create(container.getHttpHostAddress()))
+                .withRequiredParameters(FROM, SIZE)
+                .build()
+                .withRequiredStatus(PUBLISHED, PUBLISHED_METADATA)
```

Comment:
This part of test is the same in all three tests. Maybe extract to own method, like getResponse()